oak/server/rust/oak_runtime/src/node/mod.rs

```diff
@@ -206,6 +219,10 @@ impl Configuration {
             Configuration::GrpcServerNode { .. } => "GrpcServerNode".to_string(),
             Configuration::GrpcClientNode { .. } => "GrpcClientNode".to_string(),
             Configuration::WasmNode { .. } => format!("WasmNode-{}", entrypoint),
+            Configuration::StorageNode { .. } => format!("StorageNode-{}", entrypoint),
+            Configuration::RoughtimeClientNode { .. } => {
+                format!("RoughtimeClientNode-{}", entrypoint)
+            }
```

Comment:
Best to drop the -*entrypoint* suffix; as with the other pseudo-Node types, the entrypoint param is ignored for these.